lib/board.rb

```diff
@@ -0,0 +1,63 @@
+class Board
+
+  attr_accessor :cells
+
+  @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
Avoid setting instance variables outside of a method. When you instantiate a new Board instance, `cells` will not get set by this code.
```
$ Board.new.cells
# => nil
```